### DISCUSSION TITLE
Joomla! 3.9.26

### DISCUSSION THREAD

**Discussion by HLeithner** [joomla/joomla-cms#33120] [General]
New Joomla! Installations [.tar.bz2](https://github.com/joomla/joomla-cms/releases/download/3.9.26/Joomla_3.9.26-Stable-Full_Package.tar.bz2) | [.tar.gz](https://github.com/joomla/joomla-cms/releases/download/3.9.26/Joomla_3.9.26-Stable-Full_Package.tar.gz) | [.zip](https://github.com/joomla/joomla-cms/releases/download/3.9.26/Joomla_3.9.26-Stable-Full_Package.zip)
Update from Joomla! 3.9.25 [.tar.bz2](https://github.com/joomla/joomla-cms/releases/download/3.9.26/Joomla_3.9.25_to_3.9.26-Stable-Patch_Package.tar.bz2) | [.tar.gz](https://github.com/joomla/joomla-cms/releases/download/3.9.26/Joomla_3.9.25_to_3.9.26-Stable-Patch_Package.tar.gz) | [.zip](https://github.com/joomla/joomla-cms/releases/download/3.9.26/Joomla_3.9.25_to_3.9.26-Stable-Patch_Package.zip)
Update from Joomla! 3.9.x [.tar.bz2](https://github.com/joomla/joomla-cms/releases/download/3.9.26/Joomla_3.9.x_to_3.9.26-Stable-Patch_Package.tar.bz2) | [.tar.gz](https://github.com/joomla/joomla-cms/releases/download/3.9.26/Joomla_3.9.x_to_3.9.26-Stable-Patch_Package.tar.gz) | [.zip](https://github.com/joomla/joomla-cms/releases/download/3.9.26/Joomla_3.9.x_to_3.9.26-Stable-Patch_Package.zip)
Update from Joomla! 2.5 or previous 3.x releases [.tar.bz2](https://github.com/joomla/joomla-cms/releases/download/3.9.26/Joomla_3.9.26-Stable-Update_Package.tar.bz2) | [.tar.gz](https://github.com/joomla/joomla-cms/releases/download/3.9.26/Joomla_3.9.26-Stable-Update_Package.tar.gz) | [.zip](https://github.com/joomla/joomla-cms/releases/download/3.9.26/Joomla_3.9.26-Stable-Update_Package.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/joomla/joomla-cms/releases/tag/3.9.26'>Joomla! 3.9.26</a>.</em>

**Comment by ghost** [comment]
Hellow
